lib/service_actor/core.rb

```diff
@@ -11,7 +11,11 @@ module ClassMethods
     #   CreateUser.call(name: "Joe")
     def call(result = nil, **arguments)
       result = ServiceActor::Result.to_result(result).merge!(arguments)
-      new(result)._call
+
+      instance = new(result)
+      instance._call
+      instance.send(:raise_accumulated_errors) # For outputs
```

Comment:
Shouldn’t this be handled in `_call` instead ?

Reply:
In that case it won't work.